src/material/bottom-sheet/bottom-sheet-config.ts

```diff
@@ -44,7 +44,7 @@ export class MatBottomSheetConfig<D = any> {
   /**
    * Whether the bottom sheet should close when the user goes backwards/forwards in history.
    * Note that this usually doesn't include clicking on links (unless the user is using
-   * the `HashLocationStrategy`).
+   * the `HashLocationStrategy`). Defaults to `true`.
```

Comment:
We intentionally try to avoid specifying defaults in the docs, because they tend to get out of sync.

Reply:
Okay I see, thanks.

Shame TypeScript doesn't support including the default value in the dts if it's static, otherwise it could be used to generate docs.